common/djangoapps/enrollment/api.py

```diff
@@ -272,6 +275,12 @@ def update_enrollment(user_id, course_id, mode=None, is_active=None, enrollment_
         course=course_id,
         mode=mode
     ))
+
+    # [UCSD_CUSTOM] to regenerate certificates on enrollment update.
+    student = User.objects.get(username=user_id)
```

Comment:
No try except needed right?

Reply:
I think It's netter to crash server as this should never happen.